src/modules/wallet/reducer.ts

```diff
@@ -98,6 +107,25 @@ export function walletReducer(
         data: null
       }
     }
+    case FETCH_WALLET_REQUEST: {
+      return {
+        ...state,
+        loading: loadingReducer(state.loading, action)
+      }
+    }
+    case FETCH_WALLET_SUCCESS: {
+      return {
+        ...state,
+        loading: loadingReducer(state.loading, action),
+        data: action.payload.wallet
```

Comment:
Should we need to clean `error`? If so, Can we move this with `ENABLE_WALLET_SUCCESS` ?

Reply:
the `error` is not populated nor cleaned by `CONNECT_WALLET_FAILURE` either, the reason is otherwise you would see an error as soon as you arrive to the login page. The only one that produces an error is `ENABLE_WALLET_FAILURE` because that one is not expected to failed, on the other hand `CONNECT_WALLET_FAILURE` is expected to happen when the app boots and you don't have a wallet connected, which just means you need to sign in.